db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
The `related_name` for the `genres` field in the `Movie` model is set to `'genres'`. This can be confusing as it suggests a reverse relation from `Genre` to `Movie` would be accessed via `genre.genres`, which is not intuitive. Consider changing it to something more descriptive like `'movies'` to reflect the relationship more clearly.